### PR TITLE
fixes #8469 - css cleanup, whitespaces and remove dead code

### PR DESCRIPTION
**Pull request by tbrisker**
This PR updates & replaces PR #1971 which should be closed

### CODE REVIEW

File: app/assets/stylesheets/mixin.scss [hunk 1]
```diff
@@ -4,21 +4,7 @@
...
-  -ms-filter: "progid:DXImageTransform.Microsoft.gradient(startColorstr='#{$start_color}', endColorstr='#{$end_color}')";
-}
-
-@mixin border_radius($top-right, $top-left, $bottom-left, $bottom-right){
-  -webkit-border-radius: $top-right $top-left $bottom-left $bottom-right;
-  -moz-border-radius: $top-right $top-left $bottom-left $bottom-right;
-  -ms-border-radius: $top-right $top-left $bottom-left $bottom-right;
-  -o-border-radius: $top-right $top-left $bottom-left $bottom-right;
-  border-radius: $top-right $top-left $bottom-left $bottom-right;
+  background: $start_color; /* fallback for older browsers */
```
Comment (dLobatog) [inline app/assets/stylesheets/mixin.scss:7]: You might want to keep border-radius here too?

**Comment by dLobatog** [comment]
@tbrisker I want to ack because it'd be nice to merge this soon, but could you comment a bit on the changes that are not removing vendor prefixes (moz, o, etc...) or unused stuff? It's really tedious to find out what does each class do and where are they applied, sorry...

File: app/assets/stylesheets/mixin.scss (hunk 1 above)
Comment (tbrisker) [inline app/assets/stylesheets/mixin.scss:7]: `border-radius` now works unprefixed on 89.44% of all browsers (by market share), adding vendor prefixes only add 0.22% support - and even that is only in ancient browsers which are likely not used by our user base anyways. Therefor I think in favor of lesser code complexity there is no need for a mix-in. (Data about browser support and share from [here](http://caniuse.com/#feat=border-radius))

**Comment by tbrisker** [comment]
The only changes that aren't removing useless vendor prefixes or [dead code](http://i.imgur.com/UJ49zdf.jpg) are simply using SCSS to improve readability and making white-space more consistent - nothing that changes any behaviour.

File: app/assets/stylesheets/mixin.scss (hunk 1 above)
Comment (eLobato) [inline app/assets/stylesheets/mixin.scss]: The thing is it's not even unprefixed here, it was removed. I'll put it back on merge.

**Comment by dLobatog** [comment]
Merged as d109535dfdc4f0a55b2382ed24a37f67402f1c08, thanks @tbrisker !

**Pull request closed without merging**